src/login/tests/LoginPage.test.jsx

```diff
@@ -161,13 +161,14 @@ describe('LoginPage', () => {
     );
   });
 
-  it('updates the error state for invalid password', () => {
```

Comment:
any reason why we removed the invalid password test? Can we accommodate it a new test or in the test above?
`updates the error state for invalid email; no input` => `updates the error state for empty form`

Reply:
No intent of removing any tests. I guess somehow editor did that for me and i failed to recognize it on Github on a quick glance. I have again added the missing password test and changed the tests name to be more expressive.